### DISCUSSION TITLE
"DemoPlugin" plugin is not implemented on android

### DISCUSSION THREAD

**Discussion by binoysarker** [ionic-team/capacitor#4302] [Q&A]
Hello, i am new to ionic and i want to make a plugin just to test. I am following the [documentation](https://capacitorjs.com/docs/v3/plugins/workflow).  The web implementation is working, but the android is not working. I am also using NativePlugin() instead to CapacitorPlugin() because on android studio it says can not resolve CapacitorPlugin.
![image](https://user-images.githubusercontent.com/26120595/110213972-9f8a4800-7ecc-11eb-932c-3941540cc004.png)
Please help me with it. Thanks.

**Comment by jcesarmobile** [comment]
That documentation is for Capacitor v3, looks like. you are testing the plugin on a Capacitor v2 app.
